### DISCUSSION TITLE
Add cc-licence automatic badge

### DISCUSSION THREAD

**Discussion by Magicianred** [ptkdev/all-shields-cli#13] [Ideas]
I had finded this interesting repo https://github.com/santisoler/cc-licenses that indicate how include a badge for creative commons in different language.

We can add a automatism for generate cc-licence badge with config based on language and type of license. Indicate in README.md the placeholder with a comment, like the other, and without configure all the file .all-shieldsrc generate the badge

What do you think about this?

**Comment by ptkdev** [comment]
@Magicianred Great idea, i opened a issue #14. Available in release v2.1.0
